benchmarks/devel/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+add_executable(ArborX_Benchmark_Devel.exe devel.cpp)
```

Comment:
```suggestion
add_executable(ArborX_Benchmark_Devel.exe benchmark.cpp)
```

Reply:
No. I prefer filenames and targets to be aligned.